src/constants.lua

```diff
@@ -14,6 +14,14 @@ constants.minimumWithdrawalAmount = constants.mIOPerIO -- the minimum amount tha
 constants.redelegationFeeResetIntervalMs = 1000 * 60 * 60 * 24 * 7 -- 7 epochs
 
 -- ARNS
+constants.MAX_NAME_LENGTH = 51
+constants.MIN_NAME_LENGTH = 1
+-- Regex pattern to validate ARNS names:
+-- - Starts with an alphanumeric character (%w)
+-- - Can contain alphanumeric characters and hyphens (%w-)
+-- - Ends with an alphanumeric character (%w)
+-- - Does not allow names to start or end with a hyphen
+constants.ARNS_NAME_REGEX = "^%w[%w-]*%w$"
```

Comment:
This regex does not allow 1 character names!

Reply:
I think it could be:
```
^%w[%w-]*%w?$
```

Reply:
Is there not a unit test ensuring that 1 character names are allowed?

Reply:
ha, looks like that is in fact missing. will add and reasses